.gitignore

```diff
@@ -1,3 +1,4 @@
 target
 Cargo.lock
 .idea
+crates/virtio-bindings/kcov_output
```

Comment:
It would be better to specify this as just `kcov_output/` instead of using the full path because then all kcov_output directories irrespective of their parent directory will be ignored.